README.md

```diff
@@ -64,8 +69,8 @@ FreeBSD:
 Building
 ========
 
-Once the prerequisites are installed, run 'make' within the SPDK directory
-to build the SPDK libraries and examples.
+Once the prerequisites and SPDK are installed, run 'make' within the SPDK
```

Comment:
This is a little confusing; at this point, the user is trying to build SPDK, so it is not installed yet.